kolibri/plugins/coach/assets/src/views/exams-page/preview-exam-modal.vue

```diff
@@ -4,7 +4,7 @@
     <div class="question-selector-container">
       <div class="question-selector">
         <div>
-          {{ $tr('numQuestions', { num: exam.question_count })}}<icon-button :text="$tr('randomize')" @click="$emit('randomize')" :primary="true"/>
```

Comment:
To avoid completely reimplementing it - I think it might be good to use this exam modal preview there as well, but with the added randomize button? So, adding a slot for the button or somesuch?